modules/rosa-view-cloudwatch-logs.adoc

```diff
@@ -0,0 +1,22 @@
+:_module-type: PROCEDURE
+// Module included in the following assemblies:
+//
+// rosa_logging/rosa-install-logging.adoc
+
+[id="rosa-view-cloudwatch-logs_{context}"]
+= Viewing forwarded logs
+
+.Prerequisites
+
+* The `cluster-logging-operator` add-on service is installed.
+* The Amazon CloudWatch add-on service is installed.
```

Comment:
Just another hint of confusion that this looks required to be installed, but the previous procedure makes it seem like it could be optional.

Reply:
In this case it is a requirement because this is specifically for viewing the forwarded logs. I have added an intro sentence that should clarify this.